### DISCUSSION TITLE
HTML Source Help

### DISCUSSION THREAD

**Discussion by JDKamalakar** [ImranR98/Obtainium#1375] [General]
I didn't understand how to use html source for the finding apk, it would be great if anyone can help me with that.

This is the link where all the apks are: https://aniyomi.org/extensions/

Each apk have its name on left (Including its Package Name below it) on right side there is download option with version number before it.

**Comment by JDKamalakar** [comment]
Please can anyone help me out?

**Comment by ImranR98** [comment]
This site loads its content using JavaScript so Obtainium can't scrape the links.

**Reply by ImranR98** [reply]
That said, it looks like the raw info is from here: https://raw.githubusercontent.com/aniyomiorg/aniyomi-extensions/repo/index.json

So you could host your own site that presents these links in an Obtainium-friendly way.

**Comment by JDKamalakar** [comment]
Is there any documentation which I can use to make a site that hosts this data in obtinium friendly way?

**Reply by ImranR98** [reply]
Not really, you'd need to make a simple server app and then host it somewhere. Difficulty depends on whether you have any programming experience.

**Comment by DarkCrypt** [comment]
Aniyomi extensions are dead unless you've found a reliable maintainer.

**Reply by JDKamalakar** [reply]
I found it & even got update to one of the extension. (Gogo)
